Cargo.toml

```diff
@@ -25,7 +25,7 @@ path = "src/driver.rs"
 [dependencies]
 clippy_config = { path = "clippy_config" }
 clippy_lints = { path = "clippy_lints" }
-rustc_tools_util = "0.4.0"
+rustc_tools_util = { path = "rustc_tools_util", version = "0.4.0" }
```

Comment:
Please bump the version to `0.5.0`, I'll do a release before merging this PR.

Reply:
Happy to bump it to 0.5.0. The change would probably also classify as 0.4.1 being API backwards compatible. Not like version number are precious does going for 0.5.0 sounds good, just playing devils advocate that this makes room for larger changes :)

Reply:
Ah right, semver 0 versions would allow us to just bump the patch version. The `RUSTC_VERSION` addition is a new feature, normally requiring a minor version bump. But let's do 0.4.1 then.

Reply:
Bumped it to 0.4.1